MetaMask/metamask-extension
# E2e multiple erc20 watch asset

## **Description**
_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

Closes https://github.com/MetaMask/MetaMask-planning/issues/1407

This test involves handing multiple erc20 tokens on the suggested watch asset notification. Currently this tests current behavior of prod, where only the last of multiple tokens in the watchAsset confirmation will be added to the token list. With the future bump of the `@metamask/assets-controller` this test's assertion will have to change when multiple tokens are properly added.

## **Manual testing steps**

_1. Pull down this branch
_2. `yarn && yarn build:test && yarn test:e2e:single test/e2e/tests/add-multiple-tokens.spec.js --browser=${BROWSER}`
_3. Should not fail. But when bumping `@metamask/assets-controller` should fail.

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
